src/utilities/activityContent.ts

```diff
@@ -1,26 +1,41 @@
 import path from "path";
 import {
-  createVideo,
-  createImage,
   ActivityContentNote,
   ActivityContentAttachment,
+  createVideoLink,
+  createVideoAttachment,
+  ActivityContentHash,
+  createImageAttachment,
+  createImageLink,
 } from "@dsnp/sdk/core/activityContent";
 
 const createMediaAttachment = (item: string): ActivityContentAttachment => {
   const extension = path.extname(item).replace(".", "");
   const supportedVideoDomains = /youtu.be|youtube.com|vimeo.com/;
+  const activityContentHashes: Array<ActivityContentHash> = [{ algorithm: "" }];
```

Comment:
This will actually fail validation if there isn't at least one valid keccak256 content hash. There's a [`createHash` helper](https://github.com/LibertyDSNP/sdk-ts/blob/main/src/core/activityContent/factories.ts#L395) in `sdk/core/activityContent` which takes the content to hash as a string and spits out a properly formatted hash object.

Reply:
This is what was being used throughout this file, so I guess the validation changed?